.github/pull_request_template.md

```diff
@@ -8,12 +8,13 @@ Provide a concise summary of the motivation and the driving force behind this ch
 <!-- WIP: Some teams are experimenting with this section. Fee free to remove. -->
 _[Replace this with a description of the risk, if any, and how the change will be deployed.]_
 
-- [ ] Big/complex change?
-- [ ] Big splash zone?
-- [ ] High stakes if errors occur?
-- [ ] Low confidence?
-- [ ] Not hidden by feature flag?
-- [ ] Negligible risk!
+<!-- add a ✅ or 🚫 as appropriate -->
+- Big/complex change?
+- Big splash zone?
+- High stakes if errors occur?
+- Low confidence?
+- Not hidden by feature flag?
+- Negligible risk!
```

Comment:
> Just throwing out an alternate: What if the instruction were to review the list and remove any that were not risks and only keep risks that apply to your change?

like this?

```suggestion
<!-- remove any that do not apply -->
- ✅ Big/complex change?
- ✅ Big splash zone?
- ✅ High stakes if errors occur?
- ✅ Low confidence?
- ✅ Not hidden by feature flag?
- ✅ Negligible risk!
```

Reply:
I might actually like this better than my initial implementation, because deleting stuff is faster than copy-pasting the emoji

Reply:
Yes. I think only including the relevant risks and removing the rest is the clearest way to communicate them.